src/reducers/targets.js

```diff
@@ -12,55 +9,26 @@ const initialState = {
 const reducer = function (state, action) {
     if (typeof state === 'undefined') state = initialState;
     switch (action.type) {
-    case UPDATE_TARGET:
-        if (action.target.id === state.stage.id) {
-            return Object.assign({}, state, {
-                stage: Object.assign({}, state.stage, action.target)
-            });
-        }
-        return Object.assign({}, state, {
-            sprites: defaultsDeep(
-                {[action.target.id]: action.target},
-                state.sprites
-            )
-        });
     case UPDATE_TARGET_LIST:
         return Object.assign({}, state, {
             sprites: action.targets
                 .filter(target => !target.isStage)
                 .reduce(
-                    (targets, target, listId) => defaultsDeep(
+                    (targets, target, listId) => Object.assign(
                         {[target.id]: {order: listId, ...target}},
```

Comment:
Looks like `defaultsDeep` and `Object.assign` have different priority structures, where `defaultsDeep` has the objects listed in descending priority where `Object.assign` has them in increasing order of priority. e.g.

```
> a = {foo: 'bar'}
> b = {foo: 'baz'}
> require('lodash.defaultsdeep')(a, b)
{ foo: 'bar' }
> Object.assign({}, a, b)
{ foo: 'baz' }
```
So we might want to change the order of the args here?